test/index.js

```diff
@@ -271,13 +271,11 @@ test('npx with custom installer stdio', t => {
   })
 })
 
-test('noisy npx with --quiet arg on windows', {
-  skip: !isWindows && 'Only on Windows does the path to the downloaded module get printed'
-}, t => {
+test('npx with --quiet arg', t => {
   return child.spawn('node', [
     NPX_ESC, '--quiet', 'echo-cli', 'hewwo'
   ], {stdio: 'pipe'}).then(res => {
-    t.equal(res.stdout.trim(), 'hewwo')
+    t.equal(res.stdout.trim(), isWindows ? '' : 'hewwo')
     t.end()
```

Comment:
Any ideas why the `res.stdout` is empty on Windows?

Reply:
it shouldn't have been? /cc @noahleigh

Reply:
I think the switch to running in a `child` process in the current PR state has smth to do with it.